src/ci/docker/scripts/x86_64-gnu-llvm.sh

```diff
@@ -39,9 +39,6 @@ if [[ -z "${PR_CI_JOB}" ]]; then
         library/std library/alloc library/core
 fi
 
-# NOTE: intentionally uses all of `x.py`, `x`, and `x.ps1` to make sure they all work on Linux.
-../x.py --stage 2 test --skip src/tools/tidy
-
```

Comment:
this is now replaced by DOCKER_SCRIPT in CI. The order of execution changed: the equivalent of this instruction runs after the script